<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallingMenuWidget</class>
 <widget class="QWidget" name="CallingMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="LABEL_CALL_NUMBER">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_MICROPHONE_MUTE_OFF">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(30, 30, 30);
	color: rgb(70, 70, 70);
}</string>
   </property>
   <property name="text">
    <string>MIC MUTE ON</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUDIO_CLOSE">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(130, 130, 130);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>AUDIO OPENED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUDIO_OPEN">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(30, 30, 30);
	color: rgb(70, 70, 70);
}

</string>
   </property>
   <property name="text">
    <string>AUDIO CLOSED</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_PICK_UP_CALL">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	border-radius: 100px;
	background-color: rgb(0, 255, 0);
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	border: none;
	border-radius: 100px;
	background-color: rgb(0, 164, 0);
	color: rgb(164, 164, 164);
}</string>
   </property>
   <property name="text">
    <string>accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_HANG_UP_CALL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	border-radius: 100px;
	background-color: rgb(255, 0, 0);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	border: none;
	border-radius: 100px;
	background-color: rgb(164, 0, 0);
	color: rgb(164, 164, 164);
}</string>
   </property>
   <property name="text">
    <string>reject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_MICROPHONE_MUTE_ON">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(130, 130, 130);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>MIC MUTE OFF</string>
   </property>
  </widget>
  <zorder>BUTTON_AUDIO_CLOSE</zorder>
  <zorder>BUTTON_AUDIO_OPEN</zorder>
  <zorder>BUTTON_MICROPHONE_MUTE_ON</zorder>
  <zorder>LABEL_CALL_NUMBER</zorder>
  <zorder>BUTTON_MICROPHONE_MUTE_OFF</zorder>
  <zorder>BUTTON_PICK_UP_CALL</zorder>
  <zorder>BUTTON_HANG_UP_CALL</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
